content/installation/Teams/_index.md

```diff
@@ -192,11 +192,16 @@ We will use this TLS secret while deploying the BotKube backend.
                              # group/version/resource (G/V/R) format
                              # resource name should be plural
                              # (e.g apps/v1/deployments, v1/pods)
-        namespaces:          # List of namespaces, "all" will watch all the namespaces
+        namespaces:
```

Comment:
🤔 I thought we didn't want to update installation docs with 0.13 features? That was the main argumentation about merging https://github.com/kubeshop/botkube-docs/pull/112.

As I said previously, I'm fine with both approaches, as long as they are all-in:
- Have botkube.io deployed from different branch with full 0.12 documentation
- Have botkube.io deployed from main with all latest changes, even in installation docs (even with latest, as I assume the prerelease is around the corner).

But in the context of this PR I see third option:
WDYT about... removing these snippets? They are duplicates anyway, resource configurations should be in `source.md` doc and that's it.

Reply:
Yep, that's my mistake that I updated the installation doc, the should be. But I like the idea with DRY 👍 so let's remove them 🗑️